airbyte-integrations/connectors/source-salesforce/Dockerfile

```diff
@@ -1,38 +1,17 @@
-# Using alpine to remove several vulnerabilities frm slim image
```

Comment:
I reverted the changes in https://github.com/airbytehq/airbyte/pull/28021 because it was causing the build time to be 4 hours long. We should revert this if needed until the contributor can find a way to make this faster.